Scripts/RemediationScripts/Remediate-DisableBasicAuthForServiceBusNamespaces.ps1

```diff
@@ -0,0 +1,800 @@
+                    @{Expression = { $_.ResourceId }; Label = "ResourceId"; Width = 100; Alignment = "left" },
+                    @{Expression = { $_.DisableLocalAuth }; Label = "DisableLocalAuth"; Width = 100; Alignment = "left" }
+
+    if (-not $AutoRemediation) {
+        Write-Host "Azure Service Bus Namespace(s) with non-secure local authentication enabled are:"
+        $actionableServiceBusNamespacesDetails  | Format-Table -Property $colsProperty -Wrap
+        Write-Host $([Constants]::SingleDashLine)
+    }
+
+    # Back up snapshots to `%LocalApplicationData%'.
+    $backupFolderPath = "$([Environment]::GetFolderPath('LocalApplicationData'))\AzTS\Remediation\Subscriptions\$($context.Subscription.SubscriptionId.replace('-','_'))\$($(Get-Date).ToString('yyyyMMddhhmm'))\Disable-LocalAuthForServiceBusNamespaces"
+
+    if (-not (Test-Path -Path $backupFolderPath)) {
+        New-Item -ItemType Directory -Path $backupFolderPath | Out-Null
+    }
+
+    Write-Host "[Step 3 of 4] Back up Service Bus Namespace(s) details"
+    Write-Host $([Constants]::SingleDashLine)
+
+    if ([String]::IsNullOrWhiteSpace($InputFilePath)) {
+        # Backing up actionable Service Bus Namespace(s) details.
+        $backupFile = "$($backupFolderPath)\ServiceBusNamespacesDetailsBackUp.csv"
+        $actionableServiceBusNamespacesDetails | Export-CSV -Path $backupFile -NoTypeInformation
+        Write-Host "Azure Service Bus Namespace(s) details have been backed up to [$($backupFile)]" -ForegroundColor $([Constants]::MessageType.Update)
+        Write-Host $([Constants]::SingleDashLine)
+    }
+    else {
+        Write-Host "Skipped as -InputFilePath is provided" -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host $([Constants]::SingleDashLine)
+    }
+
+    if (-not $DryRun) {
+        
+        Write-Host $([Constants]::DoubleDashLine)
+        Write-Host "[Step 4 of 4] To disable Local authentication on Service Bus Namespace(s) in the Subscription" 
+        Write-Host $([Constants]::SingleDashLine)
+
+        if (-not $AutoRemediation) {
+            if (-not $Force) {
+                Write-Host "Do you want to disable local authentication on Service Bus Namespace(s) in the Subscription? " -ForegroundColor $([Constants]::MessageType.Warning)
+
+                $userInput = Read-Host -Prompt "(Y|N)"
+
+                if ($userInput -ne "Y") {
+                    Write-Host "Local authentication will not be changed for Service Bus Namespace(s) in the Subscription. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
+                    Write-Host $([Constants]::DoubleDashLine)	
+                    return
+                }
+            }
+            else {
+                Write-Host "'Force' flag is provided. Disable local authentication will be enabled on Service Bus Namespace(s) in the Subscription without any further prompts." -ForegroundColor $([Constants]::MessageType.Warning)
```

Comment:
Please see if this is more meaningful. "Disable local authentication will be enabled on Service Bus Namespace(s)" -> "Local auth will be disbaled on Service Bus Namespace(s)"